src/app/auth/auth.service.ts

```diff
@@ -45,8 +45,8 @@ export class AuthService {
     }
 
     logout() {
-        // localStorage.removeItem('LOGGED_IN');
-        // this.router.navigate(['login']);
+        this.cookieService.removeAll();
+        this.loginRedirect();
```

Comment:
have you tested this against a Universal build?

Reply:
yep 👍